### ISSUE TITLE
Error on open app

### ISSUE THREAD

**Issue by Aigor77** [swisspol/CompareFolders#31]
bash-3.2# open /Applications/Compare\ Folders.app/
LSOpenURLsWithRole() failed with error -10810 for the file /Applications/Compare Folders.app.
After purchasing  Checksum feature-

**Comment by swisspol** [comment]
Can you launch the app through the Finder? Why are you launching it through Terminal?

You should try rebooting your Mac and re-install the app from the App Store.

**Comment by swisspol** [comment]
Do you have the same problem as in #32?

**Comment by duncandisorderly** [comment]
I've got the same issue here- app was working fine, then on a whim, I decided to add the checksum feature. I'm guessing the OP above started it from terminal to get some idea of where it was failing.
error message (from a finder start)  is:

"compare folders app is damaged & can't be opened. delete compare folders app & download it from the app store"

which I've now done three times. I've been using app zapper to clear out the 'damaged' versions. it may be worth noting that the app store reports 'installed' but on a greyed-out button, which never changes to 'open' as these things normally do.

directly opening the unix executable yields this:

Last login: Wed Nov  1 18:24:10 on console
macpro:~ duncan$ /Applications/Compare\ Folders.app/Contents/MacOS/Compare\ Folders ; exit;
Failed decoding receipt field: parse date
logout

[Process completed]

any ideas?
d.
